1-exercises/B-todo-list/script.js

```diff
@@ -1,12 +1,26 @@
 function todoList(todos) {
   // Write your code here...
+  const UL = document.createElement("ul");
+  document.getElementById("content").append(UL);
+  for (let todo of todos) {
+    let li = document.createElement("li");
+    li.innerText = `${todo.todo}`;
+    li.addEventListener("click", () => {
+      if (li.classList.contains("done")) {
+        li.classList.remove("done");
+      } else {
+        li.classList.add("done");
```

Comment:
Instead of the 'add' and 'remove' methods, you can use the 'toggle' method to switch a class on and off. See more info here; https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle

Reply:
Thanks @gixo ! That's super useful to know! 👍